### ISSUE TITLE
request bin

### ISSUE THREAD

**Issue by 1mzunye-777** [titusowuor30/smartloan-mpesa-stk-push#1]
hello saw your video from github, how do i access the request bin site?

**Comment by titusowuor30** [comment]
Hi, simply provide a callback url where you want the payment data to be
sent so thta you can validate it

**Comment by 1mzunye-777** [comment]
are you on twitter? if so whats your user name?

**Comment by titusowuor30** [comment]
Yes, I am!

https://twitter.com/Titus52188294?t=AofJztoMA-YFsHfRZPSPcg&s=09

**Comment by 1mzunye-777** [comment]
bro i failed at setting stk push, can your code run live and be used on a daily basis?

**Comment by titusowuor30** [comment]
Yes, but you will need to go live. Which language are you using?

**Comment by 1mzunye-777** [comment]
java

**Comment by 1mzunye-777** [comment]
how can i send you my code, i want you to look at it?

**Comment by titusowuor30** [comment]
This is okay, we can shedule a meeting to discuss the terms. Call or
WhatsApp me here +2547 437 93 901
